cli/dts/lib.deno.unstable.d.ts

```diff
@@ -554,6 +554,9 @@ declare namespace Deno {
    * as C function pointers to ffi calls.
    *
    * The function pointer remains valid until the `close()` method is called.
+   *
+   * The callback can be explicitly referenced and dereferenced to stop Deno's
```

Comment:
I believe `dereferenced` is misleading in this context

Reply:
Used `ref'ed` and `unref'ed` for now.